src/partials/about.html

```diff
@@ -0,0 +1,28 @@
+<!-- About section (How it's made) -->
+<section class="about">
+    <div class="about__background-milk">
+            <div class="about__container about__titles">
+                <p class="about__titles--subtitle">Tradition and love</p>
+                <h2 class="about__titles--main-title">How it's made</h2>
+            </div>
+        <div class="about__flex">
+            <div class="about__container2">
+                <img class="about__container2--photo" src="images/about/eco-milk.png" alt="Bio eco cow milk"
+                    srcset="images/about/eco-milk@2x.png 2x"/> 
+            </div>
+            <div class="about__container3">
+                <p class="about__text">Fusce ut velit laoreet, tempus arcu eu, molestie tortor. Nam vel justo cursus, faucibus lorem eget, egestas eros.
+                Maecenas eleifend erat at justo fringilla.</p>
+                <p class="about__text2">Curabitur lacinia enim at ex blandit, vel pellentesque odio elementum. Mauris rhoncus orci in imperdiet placerat.
+                Vestibulum euismod nisl suscipit ligula volutpat, a feugiat urna maximus. Cras massa nibh, tincidunt.
+                
+                Aliquam erat volutpat. Aenean accumsan.</p>
```

Comment:
to ostatnie zdanie wyświetla się razem z poprzedzającym tekstem. spróbuj może < br > ?